src/modules/krita/producer_krita.yml

```diff
@@ -0,0 +1,33 @@
+schema_version: 0.3
+type: producer
+identifier: krita
+title: Krita
+version: 1
+copyright: Eoin O'Neill, Emmet O'Neill
```

Comment:
Consider adding a `description` because it is not clear at all the purpose of this producer. At a casual glance, one thinks it can read a `.kra` file to produce video frames in MLT!